modules/syslog-configuring-acs.adoc

```diff
@@ -17,5 +17,6 @@ Create a new syslog integration in {product-title}.
 . If you are using TLS, turn on the *Use TLS* toggle.
 . If your syslog receiver uses a certificate that is not trusted, turn on the *Disable TLS Certificate Validation (Insecure)* toggle.
 Otherwise, leave this toggle off.
+. Click *Add new extra field* to add extra fields. For example, if your syslog receiver accepts objects from multiple sources, type `source` and `rhacs` in the *Key* and *Value* fields, and filter on this value to identify all alerts from {product-title-short}.
```

Comment:
**nitpick:** From the example it is not clear where to filter. I think it will make sense to divide it into two sentences and state that you can filter using the custom values in your syslog receiver or SIEM.

Reply:
Updated